account_payment_order/models/account_payment_line.py

```diff
@@ -192,6 +192,7 @@ def _prepare_account_payment_vals(self):
             "date": self[:1].date,
             "currency_id": self.currency_id.id,
             "ref": self.order_id.name,
+            "name": "/",
```

Comment:
Always put comments for things that are not self-explanatory. If not, you'll have the risk that someone (or yourself) in the future removes them considering it not needed.

```suggestion
            # Put the name as the wildcard for forcing a unique name
            # If not, Odoo gets the sequence for all the payment at 
            # the same time
            "name": "/",
```